projects/project.cpp

```diff
@@ -615,6 +620,9 @@ Project* createProject() {
    if(Parameters::projectName == "Distributions") {
       rvalue = new projects::Distributions;
    }
+   if (Parameters::projectName == "ElVentana") {
+      return new projects::ElVentana;
```

Comment:
Please follow the pattern. It seems that rvalue is used for something at the bottom of this. (TBH I don't know whether that guy down there is used anywhere...)